README.md

```diff
@@ -5,25 +5,25 @@ This repo is preliminary work on a lightning network peer-to-peer node and walle
 
 It's currently being tested on Segnet4, where malleability has been solved with segwit.
 
-This version of Lnd is not yet fully-operational, but a proof of concept on Segnet4 will likely exist soon.  The projection is it will be operational before the necessary malleability soft-forks are into bitcoin mainnet (may be significantly before if there are any delays in mainnet malleability fix).
+This version of Lnd is not yet fully-operational, but a proof of concept on Segnet4 will likely exist soon.  The projection that is it will be operational before the necessary malleability soft-forks are merged into bitcoin mainnet (may be significantly before then if there are any delays in deploying the mainnet malleability fix).
```

Comment:
The first change of adding "that" doesn't make sense to me, I think it should move further back in the sentence.